apps/builder/app/routes/rest.ai.copy.ts

```diff
@@ -0,0 +1,95 @@
+import type { ActionArgs } from "@remix-run/node";
+import {
+  copywriter,
+  createGptModel,
+  type GPTModelMessageFormat,
+} from "@webstudio-is/ai";
+import { isFeatureEnabled } from "@webstudio-is/feature-flags";
+import { authorizeProject } from "@webstudio-is/trpc-interface/index.server";
+import { z } from "zod";
+import env from "~/env/env.server";
+import { createContext } from "~/shared/context.server";
+
+const RequestSchema = copywriter.ContextSchema.extend({
+  projectId: z.string(),
+});
+
+export const action = async function action({ request }: ActionArgs) {
+  if (!isFeatureEnabled("aiCopy")) {
```

Comment:
Nit we are avoiding `!i` 
```suggestion
  if (isFeatureEnabled("aiCopy") === false) {
```